subcommands/keys/ca_add_device_ca.go

```diff
@@ -60,6 +60,12 @@ func doAddDeviceCa(cmd *cobra.Command, args []string) {
 		subcommands.DieNotNil(errors.New("At least one of --online-ca or --local-ca must be true"))
 	}
 
+	if len(localCaFilename) > 0 {
+		if strings.ContainsRune(localCaFilename, os.PathSeparator) {
+			subcommands.DieNotNil(errors.New("The --local-ca-filename argument must be filename and not a path"))
```

Comment:
I did not want to enforce this limitation, but I understand your justification.

Not sure if we have similar parameters in other ca subcommands.
Can you check?

Reply:
found another spot. address in force push with a helper function.